rpa-builder/6.7/modules/ROOT/pages/_partials/oauth-credentials-properties.adoc

```diff
@@ -0,0 +1,78 @@
+//used in:
+// toolbox-mail-operations-mail-session-outlook-with-oauth
+// toolbox-variable-handling-credentials-for-oauth
+
+// tag::common-oauth-properties[]
+* *OAuth Authentication* Settings
+// tag::pin-custom-settings[]
+** *Pin OAuth Settings*
++
+Select this option to specify OAuth credentials created with the xref:toolbox-variable-handling-credentials-for-oauth.adoc[Credentials for OAuth] Action Step.
+** *Use custom settings*
++
+Select this option to specify custom OAuth credentials for this Action Step.
+// end::pin-custom-settings[]
++
+** *E-Mail Provider*
++
+Select either *Outlook* or *Google*.
+** *Auth Endpoint*
++
+The URL for the OAuth authentication endpoint. Defaults to the following values, depending on the selected provider: 
++
+*** *Outlook*: `+https://login.microsoftonline.com/common/oauth2/v2.0/authorize+`
+*** *Google*: `+https://accounts.google.com/o/oauth2/v2/auth+`
++
+** *Token Endpoint*
++
+The URL for the OAuth Refresh Token process. Defaults to the following values, depending on the selected provider: 
++
+*** *Outlook*: `+https://login.microsoftonline.com/common/oauth2/v2.0/token+`
+*** *Google*: `+https://www.googleapis.com/oauth2/v4/token+`
++ 
+** *Client ID*
++
+To authenticate with the OAuth Host, an Azure application must be defined in your space. This Azure application has an ID, that is unique in the entire OAuth Host space and it allows users to explicitly grant or revoke access for this Azure application to their accounts. For more information on how to create this Azure application for Outlook, visit https://learn.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app[Quickstart: Register an application with the Microsoft identity platform^].
```

Comment:
Is there a way to have a brief description of what this is first and then this as a separate paragraph? (For scannability and parallelism)